PropertySheets/Platform.props

```diff
@@ -25,7 +25,7 @@
 
   <ItemDefinitionGroup>
     <ClCompile>
-      <PreprocessorDefinitions>WIN32;_WIN32;_WINDOWS;WIN32_LEAN_AND_MEAN;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>WIN32;_WIN32;_WINDOWS;%(PreprocessorDefinitions)</PreprocessorDefinitions>

```

Comment:
I am not removing WIN32_LEAN_AND_MEAN, I know this is where a lot of the complications comes from having it, it is where I I had to tweak the source so that it included other headers because they now were no longer included.